index.html

```diff
@@ -104,25 +117,52 @@ <h1 style="text-align: center; ">
 community will mirror the information for the popular identifier and setup
 a working redirect to restore service to the rest of the Web.
         </p>
+        <h2 id="new">Creating a New Identifier</h2>
         <p>
-If you would like to add or update a permanent identifier to the website, the preferred
-procedure is to perform the following steps:
+If you would like to add or update a permanent identifier of the form
+<code>https://w3id.org/...</code>, the preferred procedure is to perform the
+following steps:
         </p>
         <ol>
-          <li><a href="https://github.com/perma-id/w3id.org">Fork the website</a> on Github.</li>
+          <li><em>Fork</em> <a href="https://github.com/perma-id/w3id.org">the <em>Repository</em> for this system</a> on Github.</li>
           <li>Add or update a new redirect entry and commit your changes.</li>
-          <li>Submit a pull request for your changes.</li>
+          <li>Submit a <em>Pull Request</em> for your changes.</li>
         </ol>
         <p>
-Please help out the maintainers of the service with the following:
+The maintainers of this system will then act on that <em>Pull Request</em> and merge it into this 
+system's content which you will then see but in the repository and via resolution of the identifier
+you created or edited.
```

Comment:
I can't quite puzzle out what is meant here, so I can't suggest a rewording.
```
you will then see but in the repository and via resolution of the identifier
you created or edited.
```

Reply:
Since this has already been merged, can you please open a new PR?